<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background: #3a3a3a;
}

QLabel {
	color: #fff
}

QPlainTextEdit {
	padding: 5px;
	border: 1px solid #5a5a5a;
	background-color: #5a5a5a;
	border-radius: 2px;
	color: white
}

QLineEdit {
	padding: 5px;
	border: 1px solid black;
	border-radius: 2px;
}

QToolButton, QPushButton {
	padding: 5px;
	background-color: #f4f4f4;
	border: 1px solid black;
	border-radius: 2px;
}

QToolButton:hover, QPushButton:hover {
	background-color: #909090
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Mensagem</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Mensagem Criptografada</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>400</height>
       </size>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>671</width>
        <height>61</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="placeholderText">
            <string>Chave...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Gerar Chave</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../Downloads/key_30dp_UNDEFINED_FILL0_wght400_GRAD0_opsz24.png</normaloff>../../../../../Downloads/key_30dp_UNDEFINED_FILL0_wght400_GRAD0_opsz24.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+K</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::ArrowType::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Criptografar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCriptografar"/>
    <addaction name="actionDescriptografar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionCriptografar">
   <property name="text">
    <string>Criptografar</string>
   </property>
  </action>
  <action name="actionDescriptografar">
   <property name="text">
    <string>Descriptografar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
